workers/reporter/src/test/kotlin/reporter/ReporterRunnerTest.kt

```diff
@@ -116,12 +116,12 @@ class ReporterRunnerTest : WordSpec({
             every { createTempDir() } returnsMany listOf(outputDirectory, configDirectory)
             every { close() } just runs
             coEvery { resolvePluginConfigSecrets(any()) } answers {
-                val pluginConfigs: Map<String, PluginConfiguration>? = firstArg()
+                val pluginConfigs: Map<String, org.eclipse.apoapsis.ortserver.model.PluginConfig>? = firstArg()
```

Comment:
Also here (and in other places) I guess you'd want to get rid of the FQN in favor of an import alias.